packages/react-bootstrap-table2-example/examples/columns/column-style-table.js

```diff
@@ -33,11 +33,11 @@ const columns = [{
   style: (cell, row, colIndex) => {
     if (row.id % 2 === 0) {
       return {
-        backgroundColor: '#bbe'
+        backgroundColor: '#81c784'
```

Comment:
The reason why I change demo block color is that the `main color` of our storybook is `green` including `syntax theme` and `github corner sticker`. I thought it might be better and more harmonious if we select the colors which are close to each other.  

<img width="1126" alt="2017-09-03 1 01 10" src="https://user-images.githubusercontent.com/7923460/29997356-86e9898a-9043-11e7-97c0-ea18c7b11e18.png">

Reply:
However, if you thought it became uglier, let revert to original one haha :)